Modules/Payments/Tests/Unit/PaymentMethodDeletionValidationTest.php

```
@@ -0,0 +1,136 @@
+    {
+        /** Arrange */
+        $nonExistentId = 99999;
+
+        /** Act */
+        $canDelete = $this->service->canDelete($nonExistentId);
+        $blockers = $this->service->getDeletionBlockers($nonExistentId);
+
+        /** Assert */
+        $this->assertTrue($canDelete);
+        $this->assertEquals(0, $blockers['payments']);
+    }
```

PHPMD (2.15.0)

The method it_handles_non_existent_payment_method is not named in camelCase. (undefined)

(CamelCaseMethodName)